frontend/src/components/about/AboutUs.js

```diff
@@ -96,16 +92,18 @@ class AboutUs extends React.Component {
                                 </div>
                                 <div class="card-body text-center">
                                     <h5 class="card-title">Abed</h5>
-                                    <h6 class="card-subtitle mb-2 text-muted">Junior Developer</h6>
                                 </div>
                             </div>
                         </div>
                     </div>
-                    <div className="paragraph">
-                    Lorem Ipsum is simply dummy text of the printing and typesetting industry.
-                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
-                    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
-                    It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum
+                    <div className="paragraph" style={{paddingLeft: "100px", paddingRight: "100px"}}>
```

Comment:
Did you try to use this css inside the css file for this component? i created a className "paragraph" and in the css file i added all the styling part about this paragraph. Maybe it can be useful for you